spec/lang/step/statements.md

```diff
@@ -77,10 +77,9 @@ impl<M: Memory> Machine<M> {
 
         // Write the tag directly into memory.
         // This should be fine as we don't allow encoded data and the tag to overlap for valid enum variants.
-        let accessor = |offset: Offset, value| {
+        let accessor = |offset: Offset, value_type: IntType, bytes| {
             let ptr = self.ptr_offset_inbounds(place.ptr, offset.bytes())?;
-            // We have ensured that the place is aligned, so no alignment requirement here
-            self.mem.store(ptr, [value].into_iter().collect(), Align::ONE, Atomicity::None)
```

Comment:
I think you can still use ONE here, since as before we have ensured that the place is aligned.